server/routes/api/v1/petTypeRouter.js

```diff
@@ -24,9 +25,13 @@ petTypeRouter.get("/:type", async (req, res) => {
 })
 
 petTypeRouter.get("/:type/:id", async (req, res) => {
+  console.log("inside get with findByTypeAndId method")
   try {
+    console.log(req.params.type)
     const pet = await Pet.findByTypeAndId(req.params.type, req.params.id)
     res.status(200).json({ pet: pet })
+    console.log(pet)

```

Comment:
Let's remove this log and newline :) as well as the logs on lines 18, 28, and 30!